tests/test_tokenizer.py

```diff
@@ -45,7 +45,7 @@ def test_tokenizer_against_hf(config):
             (checkpoint_dir / file).symlink_to(hf_file)
 
     theirs = AutoTokenizer.from_pretrained(
-        repo_id, cache_dir=cache_dir / "hf", local_files_only=True, token=access_token
+        repo_id, cache_dir=cache_dir / "hf", local_files_only=True, token=access_token, trust_remote_code=True
```

Comment:
```suggestion
        repo_id, cache_dir=cache_dir / "hf", local_files_only=False, token=access_token, trust_remote_code=True
```

Not sure how exactly it works inside, but I suspect that it wants to download additional files (since `trust_remove_code` is True) and limiting to only local files prevents it.